BabbleApp/babbleapp.py

```diff
@@ -93,10 +102,30 @@ async def check_for_updates(config, notification_manager):
                     f'\033[93m[{lang._instance.get_string("log.info")}] {lang._instance.get_string("babble.needUpdateOne")} [{appversion}] {lang._instance.get_string("babble.needUpdateTwo")} [{latestversion}] {lang._instance.get_string("babble.needUpdateThree")}.\033[0m'
                 )
                 await notification_manager.show_notification(appversion, latestversion, page_url)
+        except requests.exceptions.Timeout:
+            print(f'[{lang._instance.get_string("log.info")}] {lang._instance.get_string("babble.updateTimeout")}')
+        except requests.exceptions.HTTPError as e:
+            print(f'[{lang._instance.get_string("log.info")}] {lang._instance.get_string("babble.updateHttpError")}: {e}')
+        except requests.exceptions.ConnectionError:
+            print(f'[{lang._instance.get_string("log.info")}] {lang._instance.get_string("babble.noInternet")}')
         except Exception as e:
-            print(
-                f'[{lang._instance.get_string("log.info")}] {lang._instance.get_string("babble.noInternet")}. Error: {e}'
-            )
+            print(f'[{lang._instance.get_string("log.info")}] {lang._instance.get_string("babble.updateCheckFailed")}: {e}')            
```

Comment:
There is no locale string for `babble.updateCheckFailed`. This will need to be added, or you may opt to just print the error message here.